server/server.go

```diff
@@ -381,6 +381,56 @@ func (server HTTPServer) readAggregatorReportForClusterID(
 	return aggregatorResponse.Report, true
 }
 
+// readAggregatorRuleForClusterID reads report from aggregator,
+// handles errors by sending corresponding message to the user.
+// Returns report and bool value set to true if there was no errors
+func (server HTTPServer) readAggregatorRuleForClusterID(
+	orgID types.OrgID, clusterID types.ClusterName, userID types.UserID, ruleID types.RuleID, errorKey types.ErrorKey, writer http.ResponseWriter,
+) (*types.RuleOnReport, bool) {
+	aggregatorURL := httputils.MakeURLToEndpoint(
+		server.ServicesConfig.AggregatorBaseEndpoint,
+		ira_server.RuleEndpoint,
+		orgID,
+		clusterID,
+		userID,
+		fmt.Sprintf("%v|%v", ruleID, errorKey),
```

Comment:
Why to split using "|"? You can add a parameter to the endpoint

Reply:
IMHO it is based on the original design of Insights for RHEL. They choose that weird thing for some reason :1st_place_medal:

Reply:
Yes, it is based on Insights style.